src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -147,7 +147,9 @@ public abstract class Index : IDocumentTombstoneAware, IDisposable, ILowMemoryHa
         private bool _allocationCleanupNeeded;
         private readonly MultipleUseFlag _lowMemoryFlag = new MultipleUseFlag();
         private long _lastLowMemoryEventTicks = 0;
-        private readonly long _lowMemoryIntervalTicks = TimeSpan.FromMinutes(1).Ticks;
+
+        // delay a bit after low mem is over in order to let the flush(es) to finish (or at least do some major flushing writes)
+        private readonly long _lowMemoryIntervalTicks = TimeSpan.FromSeconds(1).Ticks; 
```

Comment:
Delete this, it is not required.

Reply:
It is actually needed.  We saw big perf diff without it

Reply:
Okay